runbook/main.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# ensure indices for runbooks exist
+runbook-init-storage
```

Comment:
We should split Runbooks to 2 kind of APIs: 


- Readonly API 
- Write only API
- Runbook Runner API (worker) 

Runbook Runner & Write API may have to run on site (because of security/network reasons) 
Upgrades & indexes checks should be done inside write API part 

If we split in this way we also have smooth workflow for upgrades: 
- Turn all Write API
- Run index upgrade (Run books & Read API should not be affected) 
- Turn all Runners API
- Perform upgrades for all runs data (Write API & Read API are not affected), however users can't execute runbooks for specific site
- Rolling upgrades of all read APIs